devenv/docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
       args:
         VERSION: '25.0'
     ports:
+      - 18445:18445
```

Comment:
I'd like to point out the resemblance with the mempool-api service. Are these two roughly the same?

Reply:
mempool is here because it's required by the leather wallet, but yes it's really our way of including an esplora-esque interface.